provider/maas/Makefile

```diff
@@ -8,10 +8,7 @@ bridgescript.go: add-juju-bridge.py Makefile
 	$(RM) $@
 	echo -n '// This file is auto generated. Edits will be lost.\n\n' >> $@
 	echo -n 'package maas\n\n' >> $@
-	echo -n '//go:generate make -q\n\n' >> $@
```

Comment:
Why remove this line?
Don't we need "make" in order to build this .go file to include the .py file as a string?

I suppose we don't need "import path", but I think we do want to keep the "//go:generate" line. (I *believe* it tells 'go build' to run 'make -q' to regenerate this file).